deploy/olm-catalog/dev/8.0.7-1/manifests/bamoe-businessautomation-operator.clusterserviceversion.yaml

```diff
@@ -420,7 +420,7 @@ spec:
   maturity: dev
   provider:
     name: IBM
-  replaces: bamoe-businessautomation-operator.8.0.7-4
```

Comment:
not only this file but other files need to change to correct version, please run the process again for updating it and make sure all places are updated.